plugins/tech-radar/src/components/RadarLegend/RadarLegend.tsx

```diff
@@ -32,11 +32,22 @@ export type Props = {
 };
 
 const useStyles = makeStyles<Theme>(theme => ({
+  foreignObject: {
+    overflow: 'scroll',
+    scrollbarWidth: 0 as any, // hide in FF
+    '&::-webkit-scrollbar': {
+      display: 'none', // hide in Chrome
+    },
```

Comment:
Is there any hackery to be done, to make them like one-pixel wide instead? Completely hidden can be confusing and gives no hint that there's more content to scroll to